foris/locale/de/LC_MESSAGES/foris.po

```diff
@@ -843,8 +845,8 @@ msgid ""
 msgstr ""
 "Laden Sie im folgenden Webformular die Sicherheitskopie hoch, um die "
 "Systemkonfiguration wiederherzustellen. Beachten Sie bitte, dass sich die IP-"
-"Adresse des Gerätes zwischendurch ändern kann, welches zur "
-"Nichtverfügbarkeit dieser Konfigurationsschnittstelle führen kann."
+"Adresse des Gerätes zwischendurch ändern kann, was dazu führen kann das "
```

Comment:
Shouldn't there be a comma and 'dass' (with double s)?

... was dazu führen kann, dass diese Oberfläche nicht mehr erreichbar ist.